indexer/migrations/20230523_create_schema.sql

```diff
@@ -170,8 +194,8 @@ CREATE TABLE IF NOT EXISTS l1_bridge_deposits (
     -- Deposit information
 	from_address     VARCHAR NOT NULL,
 	to_address       VARCHAR NOT NULL,
-	l1_token_address VARCHAR NOT NULL,
-	l2_token_address VARCHAR NOT NULL,
+    l1_token_address VARCHAR NOT NULL, -- REFERENCES l1_tokens(address), uncomment me in future pr
```

Comment:
The right table for these are also `l1_bridge_deposit`` and `l2_bridge_withdrawals`